### PR TITLE
chore: ☂️ Migrate InteropUiBlockListenerTest.kt to AssertJ #45610

### PR DESCRIPTION
**Pull request by Skipperlla**
## Summary:

Issue: https://github.com/facebook/react-native/issues/45596

## Changelog:

[INTERNAL] [CHANGED] - Migrated to AssertJ within file `InteropUiBlockListenerTest.kt`

## Test Plan:

Run `./gradlew -p packages/gradle-plugin test`

### CODE REVIEW

**Comment by Skipperlla** [comment]
hey @cortinico , it's done.

**Comment by Skipperlla** [comment]
hey @cortinico , it's done.

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt [hunk 1]
```diff
@@ -20,17 +20,17 @@ class InteropUiBlockListenerTest {
     val underTest = InteropUIBlockListener()
     underTest.prependUIBlock {}
 
-    assertEquals(1, underTest.beforeUIBlocks.size)
-    assertEquals(0, underTest.afterUIBlocks.size)
+    assertThat(underTest.beforeUIBlocks.size).isEqualTo(1)
```
Comment (arushikesarwani94) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt:23]: assertThat(underTest.beforeUIBlocks).hasSize(1)

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt [hunk 2]
```diff
@@ -20,17 +20,17 @@ class InteropUiBlockListenerTest {
...
+    assertThat(underTest.afterUIBlocks.size).isEqualTo(0)
```
Comment (arushikesarwani94) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt:24]: assertThat(underTest.afterUIBlocks).hasSize(0)

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt [hunk 3]
```diff
@@ -20,17 +20,17 @@ class InteropUiBlockListenerTest {
...
   }
 
   @Test
   fun addUIBlock_addsBlockCorrectly() {
     val underTest = InteropUIBlockListener()
     underTest.addUIBlock {}
 
-    assertEquals(0, underTest.beforeUIBlocks.size)
-    assertEquals(1, underTest.afterUIBlocks.size)
+    assertThat(underTest.beforeUIBlocks.size).isEqualTo(0)
```
Comment (arushikesarwani94) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/fabric/internal/interop/InteropUiBlockListenerTest.kt:32]: And similarly others to make it better.

**Review by arushikesarwani94 — CHANGES_REQUESTED** [review]

**Review by cortinico — APPROVED** [review]

**Comment by Skipperlla** [comment]
Hey @arushikesarwani94 can you approve this

**Review by arushikesarwani94 — APPROVED** [review]

**Pull request closed without merging**